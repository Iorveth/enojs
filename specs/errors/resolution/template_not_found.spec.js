const eno = require('../../../eno.js');

const input = `
copy < original
`.trim();

describe('resolution.templateNotFound', () => {

  let error;

  beforeAll(() => {
    try {
      eno.parse(input);
    } catch(err) {
      error = err;
    }
  })

  it(`provides the correct message`, () => {
    expect(error.message).toMatchSnapshot();
  });

  it(`provides correct selection metadata`, () => {
    expect(error.selection).toMatchSnapshot();
  });
});
