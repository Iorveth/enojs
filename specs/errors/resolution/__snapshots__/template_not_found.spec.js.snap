// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resolution.templateNotFound provides the correct message 1`] = `
"In line 1 the element 'original' should be copied, but it was not found.

   Line | Content
 >    1 | copy < original
"
`;
